src/emqx_mod_topic_metrics.erl

```diff
@@ -264,7 +288,12 @@ handle_call({get_rate, Topic, Metric}, _From, State = #state{speeds = Speeds}) -
                 #speed{last = Last} ->
                     {reply, Last, State}
             end
-    end.
+    end;
+
+handle_call({last_stats, LastSpeed, CurSpeed}, _From, State) ->
```

Comment:
the behavior of this clause should be the same as the `get_rate` clause.
instead of returning the lat, this new clause should return 3 values in a map.